### DISCUSSION TITLE
Hi,bros,How do I use snmp v3 as default config?

### DISCUSSION THREAD

**Discussion by lukexijn** [prometheus/snmp_exporter#1098] [General]
how to edit the default snmp.yml fro snmp v3

**Comment by lukexijn** [comment]
I run the latest snmp_ exporter from binary,not build myself

**Comment by candlerb** [comment]
I noticed that you cross-posted at https://groups.google.com/g/prometheus-users/c/u8HuBDUbRW4

In that forum, you'll find the answer to your question: https://groups.google.com/g/prometheus-users/c/l9jRsS6wg3Y
